torch/_inductor/compile_fx.py

```diff
@@ -256,18 +256,26 @@ def run(new_inputs):
 
 @dynamo_utils.dynamo_timed
 def cudagraphify(model, inputs, static_input_idxs=()):
+    from torch._inductor.cudagraph_trees import (
```

Comment:
Perhaps move all the cudagraphs related stuff (including this function) to the same file so we don't need to workaround circular imports like this.